src/components/shared/eventcard/eventcard.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Day from 'components/shared/day';
+import Box from 'components/shared/box';
+import Text from 'components/shared/text';
+import Button from 'components/shared/button';
+
+import EventPlaceholderImage from 'images/event-placeholder.png';
+
+import ClockIcon from 'images/clock.svg';
+import MapMarkerIcon from 'images/map-marker.svg';
+
+import {
+  Article,
+  ImageWrapper,
+  Image,
+  RSVPBox,
+  IconList,
+  IconRow,
+  IconText,
+  IconLink,
+} from './eventcard.css';
+
+import { parse, format } from 'date-fns';
+
+const EventCard = ({ event }) => {
+  const { name, local_date, time, featured_photo, venue, short_link } = event;
+
+  const image = () => {
+    if (featured_photo) {
+      return <Image src={featured_photo.photo_link} alt="event" />;
+    } else {
+      return <Image src={EventPlaceholderImage} alt="event" />;
+    }
+  };
+
+  const parsedDateTime = parse(time, 'T', new Date());
+  const formattedTime = format(parsedDateTime, 'h:mm a');
+
+  const venueLink = () => {
+    if (venue) {
+      const encodedAddress = encodeURI(
+        `${venue.name}, ${venue.address_1}, ${venue.city}, ${venue.state}`
+      );
+      const directionsUrl = `https://www.google.com/maps/search/?api=1&query=${encodedAddress}`;
+      return (
+        <IconRow mt={3}>
+          <Box flex="none">
+            <MapMarkerIcon />
+          </Box>
+          <IconLink
+            as="a"
+            href={directionsUrl}
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            {venue.name}
+          </IconLink>
+        </IconRow>
+      );
+    }
+    return null;
+  };
+
+  const handleRSVPClick = () => window.open(short_link);
```

Comment:
Is this used?

Reply:
Nope, sorry about that. Removed in 71aaeb5.